drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -168,8 +168,8 @@ static void bt_ipm_rx_thread(void)
 				/* Vendor events are currently unsupported */
 				BT_ERR("Unknown evtcode type 0x%02x",
 				       hcievt->evtserial.evt.evtcode);
-				k_sem_give(&ipm_busy);
-				break;
+				TL_MM_EvtDone(hcievt);
```

Comment:
It looks like this wasn't here earlier, but it would make sense that it should be.

Reply:
indeed, this was an issue in the initial version.